test/unit/data/custom-metadata-json/happy_flow_app_android.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+    "metadata": [
```

Comment:
I think this can be removed, I also think the `App` needs to be removed from the testdata so we have "clean" data

Reply:
I cleaned out the App because it indeed didn't make sense and was confusing.